.github/workflows/elixir.yml

```diff
@@ -38,24 +38,25 @@ jobs:
           --health-retries 5
     steps:
       - uses: erlef/setup-beam@v1
+        id: setup-beam
         with:
-          otp-version: "26.0.2"
-          elixir-version: "1.15.2"
+          otp-version: "26.1.1"
+          elixir-version: "1.15.6"
       - uses: actions/checkout@v4
       - uses: actions/cache@v3
         name: Elixir Deps Cache
         env:
           cache-name: cache-elixir-deps-${{ env.MIX_ENV }}
         with:
           path: elixir/deps
-          key: ${{ runner.os }}-${{ steps.setup-beam.outputs.elixir-version }}-${{ steps.setup-beam.outputs.otp-version }}-${{ env.cache-name }}-${{ hashFiles('**/elixir/mix.lock') }}
```

Comment:
This is redundant, `elixir-version` output from step above includes the otp version it turns out